### PR TITLE
chore: update simnet images

### LINKED ISSUE #326: update xud-simnet
- scripts seem to be outdated (peers.sh connecting to non-existent lndbtc/lndltc peers)
- lndltc neutrino & binary updated (https://github.com/ExchangeUnion/xud-simnet/pull/67)
- lndbtc binary updated (https://github.com/ExchangeUnion/xud-simnet/pull/71)

### PR DESCRIPTION
**Pull request by reliveyy**
This commit will update simnet environment to [latest xud-simnet master](https://github.com/ExchangeUnion/xud-simnet/tree/5d6835a189c42f2c9a95cec366fbcd55b3abc050)

**How to test?**

Run xud script with `-b chore/update-simnet` then select simnet you should have latest xud-simnet experience (auto channel openning is included in this PR).

Closes #326
Reopen previous PR https://github.com/ExchangeUnion/xud-docker/pull/346

### CODE REVIEW

File: tests/integration/test_simnet.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+                break
+        return target
+
+    utils = find("utils")
+
+    if utils:
+        exit_code, output = utils.exec_run("cat /var/log/launcher.log")
+        print(output.decode())
+
+    # exit_code, output = find("xud").exec_run("xucli --rpcport=18886 getinfo")
```
Comment (erkarl) [inline tests/integration/test_simnet.py:102]: Why comment and not delete?

Comment (reliveyy) [reply tests/integration/test_simnet.py:102]: This is the part I've not planned well about which is revealing some useful debug information when the test fails. Anyway I well refine this part later.

**Comment by peartobear** [comment]
A more readable error state would be better (?)

Screenshot 2020-04-13 at 2 55 00 PM

**Comment by reliveyy** [comment]
> A more readable error state would be better (?)

Sorry this PR is still pending becuase the our GCP simnet is still not ready.

```
❯ docker exec simnet_xud_1 xucli --rpcport 28886 connect 02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6@xud1.simnet.exchangeunion.com:28885
Error: 2 UNKNOWN: Peer 02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6@xud1.simnet.exchangeunion.com:28885 was destroyed
```

**Comment by kilrau** [comment]
This is currently not fully working due to https://github.com/ExchangeUnion/xud-docker/pull/395

```
simnet > status
┌─────────┬──────────────────────────────────────────────────┐
│ SERVICE │ STATUS                                           │
├─────────┼──────────────────────────────────────────────────┤
│ lndbtc  │ Ready                                            │
├─────────┼──────────────────────────────────────────────────┤
│ lndltc  │ Ready                                            │
├─────────┼──────────────────────────────────────────────────┤
│ raiden  │ Container exited                                 │
├─────────┼──────────────────────────────────────────────────┤
│ xud     │ could not connect to xud at 0.0.0.0:28886, is xu │
└─────────┴──────────────────────────────────────────────────┘
simnet > getinfo
could not connect to xud at 0.0.0.0:28886, is xud running?
```

**Comment by kilrau** [comment]
Last commit sets xud grpc listening address to 127.0.0.1, which magically turned the xud grpc listen on `localhost` instead of `::1` and `xucli` is working again:

```
bash-5.0# netstat -a
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       
tcp        0      0 0.0.0.0:28885           0.0.0.0:*               LISTEN      
tcp        0      0 0.0.0.0:28887           0.0.0.0:*               LISTEN      
tcp        0      0 localhost:9050          0.0.0.0:*               LISTEN      
tcp        0      0 127.0.0.11:46657        0.0.0.0:*               LISTEN      
tcp        0      0 localhost:54764         localhost:9050          ESTABLISHED 
tcp        0      0 localhost:28885         localhost:47438         ESTABLISHED 
tcp        0      0 xud:42842               lndltc:10009            ESTABLISHED 
tcp        0      0 localhost:47438         localhost:28885         ESTABLISHED 
tcp        0      0 localhost:9050          localhost:54764         ESTABLISHED 
tcp        0      0 localhost:9050          localhost:54668         ESTABLISHED 
tcp        0      0 xud:42772               lndbtc:10009            ESTABLISHED 
tcp        0      0 localhost:54668         localhost:9050          ESTABLISHED 
tcp        0      0 xud:45698               45.62.228.131:9001      ESTABLISHED 
tcp        0      0 localhost:28886         :::*                    LISTEN      
udp        0      0 127.0.0.11:58132        0.0.0.0:*                      
```

**Comment by kilrau** [comment]
This [commit](https://github.com/ExchangeUnion/xud-docker/pull/385/commits/f6d6903aca7b009f848a9007d96e18c5ce7c4055) moving all `latest` images together broke the auto create and auto unlock of xud.

We could now try to bring things back, but I rather suggest we remove all simnet specific logic and have the regular create/write down mnemonic, set passwords, set backup location flow.

**Comment by raladev** [comment]
it was decided to create separate issue for task in priveous comment, since it was easier to return autocreation and we did not want to block the merge by this comment. #425

**Comment by raladev** [comment]
**Got**:
Error: 13 INTERNAL: could not initialize lnd-LTC: 14 UNAVAILABLE: failed to connect to all addresses
Error: 13 INTERNAL: could not initialize lnd-BTC: 14 UNAVAILABLE: failed to connect to all addresses

Steps:
1. Install xud-docker for mainnet network with light setup
2. Try create wallet

Screens:
installation output - [Screenshot from 2020-04-22 12-31-52](https://user-images.githubusercontent.com/29906866/79966401-0dd14200-8496-11ea-9e5d-f6c514208dae.png)
versions - [Screenshot from 2020-04-22 12-33-47](https://user-images.githubusercontent.com/29906866/79966294-e8443880-8495-11ea-9fd7-8f8fc8446128.png)
xud logs - [Screenshot from 2020-04-22 12-32-02](https://user-images.githubusercontent.com/29906866/79966332-f85c1800-8495-11ea-8401-65f2342feef2.png)

**Pull request merged**